includes/admin/class-llms-admin-header.php

```diff
@@ -54,30 +55,38 @@ public function admin_header() {
 			$show_header = true;
 		}
 
+		// Get the current page if available.
+		$page = sanitize_text_field( wp_unslash( $_GET['page'] ?? '' ) );
```

Comment:
```suggestion
		$page = llms_filter_input( INPUT_GET, 'page' ); 
```

should be enough... and we can remove all the phpcs:disable below?